warehouse/query-core/src/test/java/datawave/query/jexl/JexlNodeFactoryTest.java

```diff
@@ -0,0 +1,192 @@
+package datawave.query.jexl;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import datawave.query.jexl.lookups.IndexLookupMap;
+import datawave.query.jexl.visitors.JexlStringBuildingVisitor;
+import org.apache.commons.jexl2.parser.ASTEQNode;
+import org.apache.commons.jexl2.parser.ASTERNode;
+import org.apache.commons.jexl2.parser.ASTJexlScript;
+import org.apache.commons.jexl2.parser.JexlNode;
+import org.apache.commons.jexl2.parser.ParseException;
+import org.apache.commons.jexl2.parser.ParserTreeConstants;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class JexlNodeFactoryTest {
+    
+    // It's worth noting that 'expandFields' is almost always set to true. This test is a contrived case that exercises a code path.
+    @Test
+    public void testNoFieldExpansion() throws ParseException {
+        IndexLookupMap indexLookupMap = new IndexLookupMap(10, 10);
+        indexLookupMap.putAll("FOO", Sets.newHashSet("6", "9"));
+        indexLookupMap.putAll("BAR", Sets.newHashSet("7", "8"));
+        
+        JexlNode originalNode = JexlASTHelper.parseJexlQuery("((_Bounded_ = true) && (FOO > 5 && FOO < 10))");
+        JexlNode node = JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE, false, originalNode, indexLookupMap, false,
```

Comment:
Just curious, there be a practical way to throw in a unit test that goes through the cases where ContainerType.AND_NODE is used?

Reply:
Added test in https://github.com/NationalSecurityAgency/datawave/pull/1383/commits/753ec82e172a2f8ecce0be3fe7e14ee5705f78f2

Reply:
Cool, thank you.